test/occupation.jl

```diff
@@ -134,3 +134,28 @@ if mpi_nprocs() == 1 # can't be bothered to convert the tests
     end
 end
 end
+
+if mpi_nprocs() == 1 # can't be bothered to convert the tests
+@testset "Fixed Fermi level" begin
+    testcase = magnesium
+    Ecut = 5
+    fft_size = [15, 15, 15]
+    kgrid  = [2, 3, 4]
+
+    parameters = (
+        (DFTK.Smearing.FermiDirac(), 0.01, 0.2898048739348472, ≈),
```

Comment:
If smearing and temperature are the same, don't put them here. Also make a comment about the 0.28 ....

Reply:
In fact I would actually just compute it in the test.